x-pack/plugins/observability_solution/inventory/public/components/alerts_badge/alerts_badge.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import rison from '@kbn/rison';
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Entity } from '../../../common/entities';
+import { useKibana } from '../../hooks/use_kibana';
+import { parseIdentityFieldValuesToKql } from '../../../common/utils/parse_identity_field_values_to_kql';
+
+export function AlertsBadge({ entity }: { entity: Entity }) {
+  const identityFieldValues = parseIdentityFieldValuesToKql({ entity });
+  const {
+    services: {
+      http: { basePath },
+    },
+  } = useKibana();
+
+  const activeAlertsHref = basePath.prepend(
+    `/app/observability/alerts?_a=${rison.encode({
+      kuery: identityFieldValues.join(' AND '),
```

Comment:
Does it make sense to move this to `parseIdentityFieldValuesToKql`?

Reply:
I believe it does, thanks for the suggestion

Reply:
https://github.com/elastic/kibana/pull/195250/commits/638c8515f6562f8dca2d78e3d71215233cb5406e